packages/expo/package.json

```diff
@@ -23,11 +23,24 @@
   },
   "license": "MIT",
   "author": "Clerk",
-  "main": "dist/index.js",
-  "source": "src/index.js",
-  "typings": "dist/index.d.ts",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/index.js",
```

Comment:
We're only outputting CJS so you should combine `import` and `require` into `default`.

If we then ever publish CJS + ESM, we can split it into both again

Reply:
Fixed!